js/content/community.js

```diff
@@ -858,6 +878,58 @@ let ProfileHomePageClass = (function(){
     return ProfileHomePageClass;
 })();
 
+
+let GroupHomePageClass = (function(){
+
+    function GroupHomePageClass() {
+        this.addExtraLinks();
+    }
+
+    GroupHomePageClass.prototype.addExtraLinks = function() {
+
+        let groupId = GroupID.getGroupId();
+        let iconType = "none";
+        let images = SyncedStorage.get("show_profile_link_images");
+        if (images !== false) {
+            iconType = images === "color" ? "color" : "gray";
+        }
+
+        let links = [
+            {
+                "id": "steamgifts",
+                "link": `//www.steamgifts.com/go/group/${groupId}`,
+                "name": "SteamGifts",
+            }
+        ];
+
+
+        // Build the links HTML
+        let htmlstr = "";
+
+        links.forEach(link => {
+            if (!SyncedStorage.get("group_" + link.id)) { return; }
+            htmlstr +=
+                `<div class="es_profile_link profile_count_link">
```

Comment:
If you're adding the `weblink` class here, hovering over it will also color it like the other links on the right side.
You'd need to probably change the selector `.es_sites_icons.es_gray` to `.es_sites_icons.es_gray:after`, haven't tested it that much though.

If you don't want to do this, I can after merging

Reply:
I don't understand how to do this, I suck at css, sorry.

Reply:
Added with e3f40ba1c5c4dcf6c05b285c27738b4040ba5528